tests/packagemanager_deb_tests.cc

```diff
@@ -0,0 +1,47 @@
+#include <gtest/gtest.h>
+
+#include <boost/filesystem.hpp>
+#include <boost/shared_ptr.hpp>
+
+#include <config.h>
+#include <packagemanagerfactory.h>
+#include <packagemanagerinterface.h>
+#include <utils.h>
+
+TEST(PackageManagerFactory, Debian_Install_Good) {
+  Config config;
+  config.pacman.type = kDebian;
+  boost::shared_ptr<PackageManagerInterface> pacman =
+      PackageManagerFactory::makePackageManager(config.pacman, boost::filesystem::path("/tmp/"));
```

Comment:
TemporaryDirectory (in utils.h) would be better here, because writing to fixed locations in /tmp is likely to cause trouble with parallel tests, and can cause problems because the old test runs get left around.  If we just use a fresh TemporaryDirectory every time, then there is no problem with junk getting left around and breaking tests